tap_salesforce/__init__.py

```diff
@@ -73,9 +75,10 @@ def build_state(raw_state, catalog):
             job_id = singer.get_bookmark(raw_state, tap_stream_id, 'JobID')
             batches = singer.get_bookmark(raw_state, tap_stream_id, 'BatchIDs')
             current_bookmark = singer.get_bookmark(raw_state, tap_stream_id, 'JobHighestBookmarkSeen')
+            adjusted_current_bookmark = singer_utils.strftime(singer_utils.strptime_with_tz(current_bookmark) - datetime.timedelta(seconds=int(CONFIG.get('lookback_window', 10))))
```

Comment:
Can we make a small function of it? It will increase the readability.

Reply:
Updated the code.